tests/main/snap-advise-command/task.yaml

```diff
@@ -2,17 +2,23 @@ summary: Ensure that `snap advise-snap` works
 
 # we need https://github.com/snapcore/core/pull/70 landed before
 # we can use this on ubuntu-core-*
-systems: [ubuntu-16-*, ubuntu-18-*]
+systems: [ubuntu-16*, ubuntu-18*]
 
 prepare: |
-    mv /usr/lib/command-not-found /usr/lib/command-not-found.orig
+    if [ -e /usr/lib/command-not-found ]; then
+        mv /usr/lib/command-not-found /usr/lib/command-not-found.orig
+    fi
 
 restore: |
     if [ -e /usr/lib/command-not-found.orig ]; then
         mv /usr/lib/command-not-found.orig /usr/lib/command-not-found
     fi
 
 execute: |
+    # FIXME: remove this once the store is in good shape again
```

Comment:
afaict this will need tweaks once we enable it again, it seems some things it is checking have changed since it was running